content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+
+## Wrinkle Mask Extension
+It is important that the masks use the _wrinklemask suffix and .tif extension, since the model builder currently looks for the _wrinklemask.tif extension to see if a mask that corresponds with the morph target exists. The suffix of _wrinklemask also ensures that the textures are processed and compressed appropriately by the asset processor using the correct settings from the _wrinklemask texture preset.
+
+## Wrinkle Mask File Properties
+The bit depth and resolution of the textures should not impact the functionality of the skin shader, so use whatever settings match the fidelity you need. We have only tested with 32 bits per channel, 4k textures. 4k is almost certainly excessive for a mask. 512x512 seems like it would be plenty, but it's at your discretion. You should be able to use 8 bits per channel while still having sufficient precision.
+
+## Mask Blending
+It is possible for multiple overlapping morph targets to be active at once. In this case, the contribution of each mask will be summed.
+
+As an example, let's say that for the left_eyebrow_raised morph target, the portion of the forehead directly above the left eyebrow is fully wrinkled using the first wrinkle layer. This means that portion of the mask will be bright red, indicating that the first wrinkle layer should be used for the normal. For the portion of the mask that covers the center of the forehead, the mask might be a darker, mid-range red that indicates that the middle of the forehead is only halfway to fully wrinkled when the left eyebrow is raised.
+
+The right_eybrow_raised mask might be a mirror or nearly-mirror image of the left mask, with the center of the forehead also halfway wrinkled. But if both the left eyebrow and right eyebrow are fully raised at the same time, the overlapping values in the center of the forehead from the two masks will be added together, resulting in the center of the forehead being fully wrinkled using the first wrinkle layer.
+
+When the values from multiple masks are combined, they will be clamped in the 0-1 range, so they will never exceed 1 for any given layer.
+
+## Corrective Shapes
+It's not clear if corrective shapes will need their own mask. If they are just being used to smooth out the transition between two other shapes, the combined mask values from the other shapes may be enough. Since the blend masks are only additive, a corrective shape will not be able to reduce the influence of a wrinkle layer, only add to it.
```

Comment:
- Beginner question, or a reminder to users, what's a corrective shape? I suggest adding this to the section.
- This should also be updated to report on the feature as it currently is. So either removing or amending the first two sentences.

Reply:
We can remove this from here then. You can add the the issues to the known issues for the upcoming release notes. I don't think known issues necessarily need to be only for features being released. It's possible to find or callout issues in a future release. 
```suggestion
```